src/main/java/io/github/christophermanahan/carnitas/HTTPServer.java

```diff
@@ -29,7 +29,9 @@ private Runnable connect(Listener listener) {
 
     private void serve(Connection connection) {
         parser.parse(new ConnectionReader(connection))
+          .map(logger::log)
           .map(handler::handle)
+          .map(logger::log)
```

Comment:
Let's think about how we could simplify this.

Hint: Think about logging as something that the developer could configure (i.e. making it part of the application).

Hint: Middleware 😄